micro-frontends/webpack.config.js

```diff
@@ -0,0 +1,136 @@
+const path = require("path");
+const cssExtract = require("mini-css-extract-plugin");
+const ModuleFederationPlugin = require("webpack/lib/container/ModuleFederationPlugin");
+
+const packageJson = require("./package.json");
+const dependencies = packageJson.dependencies;
+
+module.exports = {
+  resolve: {
+    extensions: [".tsx", ".ts", ".jsx", ".js", ".json"],
+    alias: {
+      react: path.resolve(__dirname, "./src/globalReact.js"),
+      'react-dom': path.resolve(__dirname, "./src/globalReactDom.js"),
+    }
+  },
+  entry: {
+    ipd: "./src/ipd/index.js",
+    mfe_polyfills_angular_1_4: "./src/polyfill.js",
```

Comment:
@AshishkurianTw I remember in the earlier PR `"next-ui": "./src/next-ui/index.js",
    shared: "./src/shared.js"` was added to the entry instead of `mfe_polyfills_angular_1_4`. Just wanted to understand if this change was deliberate and the reasoning behind it.